src/codemagic/models/xcodebuild.py

```diff
@@ -53,9 +58,11 @@ def _get_logs_path(self) -> pathlib.Path:
             tmp_dir = pathlib.Path(tempfile.gettempdir())
         logs_dir = tmp_dir / 'xcodebuild_logs'
         logs_dir.mkdir(exist_ok=True)
-        prefix = f'{self.xcode_project.stem}_'
-        with tempfile.NamedTemporaryFile(prefix=prefix, suffix='.log', dir=logs_dir) as tf:
-            return pathlib.Path(tf.name)
+        path_candidates = itertools.chain(
```

Comment:
Is this change related?

Reply:
Not really. But it helped a great deal with testing everything out locally. You know the previous way of logfile generation resulted in random suffixes in file name that could not be easily sorted without taking file creation time into account.